### ISSUE TITLE
[Feature Request]: Make Employee Number searchable

### ISSUE THREAD

**Issue by nastybutler** [grokability/snipe-it#11963]
### Is your feature request related to a problem? Please describe.

I fully expected this to be something everyone would want and ask for.  We have so many laborers working that inevitably similar names crop up.  Looking for John A Smith as opposed to John T Smith to the nth degree gets really old.  Imported employee numbers only to discover they are not a searchable parameter.

### Describe the solution you'd like

Make the employee number field a searchable item.  Please!

### Describe alternatives you've considered

Our alternative right now is to look up all the users of that name and then spot check the employee number.  two steps when it could be one.

**Comment by brianhoganm** [comment]
I totally agree. Having several thousand students...this would be very useful! 👍

**Comment by snipe** [comment]
This was added a long time ago I think.
